Sources/SwiftFrameCore/Workers/ImageWriter.swift

```diff
@@ -0,0 +1,103 @@
+import AppKit
+import Foundation
+
+public final class ImageWriter {
+
+    // MARK: - Exporting
+
+    static func finish(
+        context: CGContext,
+        with outputPaths: [FileURL],
+        sliceSize: CGSize,
+        outputWholeImage: Bool,
+        locale: String,
+        suffix: String,
+        format: FileFormat,
+        completion: @escaping () throws -> Void) throws {
```

Comment:
Micro-Nit: `{` on the next line makes it easier to separate signature and body IMHO